uwblueprint/project-read-frontend
# Notes field

### Notion ticket link



[Notes section](https://www.notion.so/uwblueprintexecs/Notes-section-694b94f92367427fb0852ebf5fe0c095)



### Implementation description

- Add notes text field in Family Details Sidebar



### Steps to test

Backend PR: https://github.com/uwblueprint/project-read-backend/pull/70
1. Verify each family's notes displays in the sidebar



### What should reviewers focus on?

- Clean, robust code & correctness

### Checklist
